oaknational/Oak-Web-Application
# Add `<IconButtonFocusVisible/>` to allow `:focus-visible` instead of `:focus`

## Description
Changing `:focus` to `:focus-visible` in `<IconFocusUnderline/>` which effects all the core components.

## Issue(s)

Fixes `CUR-973`

## How to test
1. Go to https://deploy-preview-2921--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-secondary-aqa/units
2. Open a unit modal using a mouse and check for lack of focus outline on the close button.
3. Close the modal.
4. Open a unit modal using a keyboard and check that the focus outline on the close button is still present.
5. Repeat steps 2-4 using an optionality modal.
